### ISSUE TITLE
Update of access rights requires restart

### ISSUE THREAD

**Issue by hopfenzapfen** [Requarks/wiki#84]
### Actual behavior
If I am not mistaken, changes to the user's access rights are only taking effect after restarting wiki with `node wiki.js stop && node wiki.js start`. This is not not a big issue, when its possible to log on to the server. But when the wiki is used and administered by other people that cannot SSH into the server, it is quite annoying.

### Expected behavior
Update changes to access rights without requiring a manual restart.

### Steps to reproduce the behavior
F.e. when denying access for the guest user.

**Comment by NGPixel** [comment]
Rights are applied as soon as you hit saved. You don't need to restart the server at all.

If you don't want public guest access, you should simply disable `public` option in the config.yml file instead.

**Comment by hopfenzapfen** [comment]
I already know about the `public` option in the `config.yml` file. Thanks. I have it set to `true`.
The exact problem I am facing is the following.
There is one subpage of my wiki (the `/aboutme` path) that I allowed and denied guest access to, a few times. Every time I changed the access from denied to allowed or vice versa, the access rights were not enforced after clicking safe.
So, for example when changing access from allowed to denied, I could still access that page from another browser from which I wasn't logged in. I only saw the "Forbidden" page after restarting the wiki and reloading the page.
Will the access rights be enforced immediately after updating them or do I just need to be more patient and wait for a while until they are taking effect? I only waited for 1-2 minutes.
